core/tests/protractor.conf.js

```diff
@@ -56,6 +56,7 @@ exports.config = {
   // When run without a command line parameter, all suites will run. If run
   // with --suite=smoke, only the patterns matched by that suite will run.
   suites: {
+
```

Comment:
I don't know what you did, but you can fix it by modifying it back to what it used to be and adding another commit to this PR. After you push a PR, take a look at the "Files Changed" tab in the PR on GitHub to make sure that the diff is exactly what you want it to be.